Jenkinsfile

```diff
@@ -128,7 +128,7 @@ def androidUnitTests(testName, nodeVersion, npmVersion, deviceId) {
 
 def macosUnitTests(nodeVersion, npmVersion) {
 	return {
-		node('git && xcode-12') {
+		node('git && xcode-13') {
```

Comment:
We currently only have 1 Xcode 13 node, so this build will take a little while. We'll roll Xcode 13 to another 2 soon

Reply:
We now have Xcode 13 on 3 nodes